resources/sections.md

```diff
@@ -1,17 +1,17 @@
 ---
 title:  "Sections - Resources - Western Region, Order of the Arrow, Boy Scouts of America"
-date:   2016-04-29 17:27 -0600
+date:   2017-04-12 20:10 -0800
 description: "Resources for Sections in the Western Region."
 ---
 
 ## Conclave Insurance Payment Transmittal Form
 
 The transmittal paperwork for the annual Section Conclave's insurance payment. This is a fillable PDF.
 
-<a href="{{ site.baseurl }}resources/assets/2017-conclave-insurance-payment-transmittal-fillable.pdf" class="btn btn-default"><i class="fa fa-file-pdf-o"></i> Download Insurance Payment Transmittal Form</a>
+<a href="{{ site.baseurl }}resources/assets/2017-Conclave-Insurance-Payment-Transmittal-Fillable.pdf" class="btn btn-default"><i class="fa fa-file-pdf-o"></i> Download Insurance Payment Transmittal Form</a>
```

Comment:
I think the Linux server that we are on is case-sensitive. Please adjust the case to match the file.